src/frameworks/TYPO3/faq.md

```diff
@@ -0,0 +1,22 @@
+# TYPO3 Frequently Asked Questions (FAQ)
```

Comment:
I know other frameworks still have them, but from an IA standpoint I really hate FAQs. :smile: If something is so frequently asked, it should be a normal part of the documentation.